core-bundle/contao/languages/en/tl_page.xlf

```diff
@@ -356,6 +356,30 @@
       <trans-unit id="tl_page.twoFactorJumpTo.1">
         <source>Please choose the page to which members will be redirected to set up two-factor authentication.</source>
       </trans-unit>
+      <trans-unit id="tl_page.enableCsp.0">
+        <source>Enable CSP</source>
+      </trans-unit>
+      <trans-unit id="tl_page.enableCsp.1">
+        <source>Enable Content Security Policy for this domain.</source>
```

Comment:
This isn‘t correct, is it? CSP is only enabled per website root page but if I enable it for `en`, it is not automatically enabled for `de` as well. So it is not enabled for "this domain".

Reply:
Changed in 23a441d99c